data/mode.txt

```diff
@@ -1 +1 @@
-secret
+extended
```

Comment:
I have no idea what is going on with my PR right now to be honest but I think that's a text file that lists the local computer's current hosted game mode